### DISCUSSION TITLE
Vaultwarden Alpine Linux

### DISCUSSION THREAD

**Discussion by xdxxp6qp** [dani-garcia/vaultwarden#3146] [General]
hello, let me know.  I saw in the Alpine Linux distribution repository.  apk vaultwarden.  do you have anything to do with this, is it your developers who spread it?

**Comment by BlackDex** [comment]
Looking at the APKBUILD file, it's not from one of the main developers/contributors.
https://git.alpinelinux.org/aports/tree/testing/vaultwarden/APKBUILD

But, that shouldn't matter that much. There is also a Arch Linux version of Vaultwarden, also not maintained by one of the main developers/contributors.

**Comment by xdxxp6qp** [comment]
I'm wondering how safe it is to use?

**Reply by BlackDex** [reply]
Looking at the APKBUILD file, it downloads the source from this repo, and then builds it.

You can see all the build logs, like this one from armv7 for example: https://build.alpinelinux.org/buildlogs/build-edge-armv7/testing/vaultwarden/vaultwarden-1.27.0-r0.log on there site.

**Reply by xdxxp6qp** [reply]
I don't understand anything in the log.  but you say it's safe?

**Reply by BlackDex** [reply]
Looking at the APKBUILD file they do not change anything to the code, and they download the code directly from github from this repo's releases. So, from there i conclude that it is safe to use.

Arch Linux has something similar, and people are using that also.

**Reply by xdxxp6qp** [reply]
thank you very much
